rmgpy/tools/canteramodel.py

```diff
@@ -208,24 +221,39 @@ def __init__(self, species_list=None, reaction_list=None, canteraFile='', output
         `sensitive_species`: a list of RMG species objects for conductng sensitivity analysis on
         `thermo_SA`: a boolean indicating whether or not to run thermo SA. By default, if sensitive_species is given,
                       only kinetic_SA will be calculated and it must be additionally specified to perform thermo SA.
+        `surface`: a boolean indicating whether or not to run a surface simulation
```

Comment:
Please add docstring entries for surface_species_list and surface_reaction_list as well.